src/main/java/org/opentripplanner/standalone/config/NodeAdapter.java

```diff
@@ -93,7 +93,7 @@ public List<NodeAdapter> asList() {
     for (JsonNode node : json) {
       String pName = "[" + i + "]";
       NodeAdapter child = new NodeAdapter(node, source, fullPath(pName));
-      children.add(child);
+      childrenByName.put(pName, child);
```

Comment:
Can this `pName` parameter be renamed? I don't even know what the p stands for but my best guess is param.